sidecar/resources.py

```diff
@@ -182,14 +181,13 @@ def _update_file(data_key, data_content, dest_folder, metadata, resource,
 def _watch_resource_iterator(label, label_value, target_folder, url, method, payload,
                              current_namespace, folder_annotation, resource, unique_filenames, script, enable_5xx):
     v1 = client.CoreV1Api()
-    namespace = os.getenv("NAMESPACE", current_namespace)
     # Filter resources based on label and value or just label
     label_selector = f"{label}={label_value}" if label_value else label
 
-    if namespace == "ALL":
+    if current_namespace == "ALL":
```

Comment:
I don't think `current_namespace` can be `ALL` at this point since it is not a user provided argument but the auto detected *current* namespace.